Makefile

```diff
@@ -36,6 +39,10 @@ PRINT_FLOAT ?= off
 # MAKECMDGOALS is a Make variable that is set to the target your building for.
 TARGET = $(MAKECMDGOALS)
 
+ifeq ($(TARGET), linux)
+	SNAPSHOT = off
+endif
```

Comment:
This block doesn't catch the "make BOARD=linux" case. (See line 58.) That will still try to build with snapshot.